index.html

```diff
@@ -1,77 +1,115 @@
-        
-    </ul>
-</div>
-
-</section>
-
-<section>
-<h1>Connect </h1>
-<a href= https://www.linkedin.com/in/kipardillo/> linkedin </a>
-</section>
-
-<section id = "contact">
-    <h2> Leave a Message</h2>
-    <form name = "leave_message">
-       
-        <label for  = leave_message> </label>
-
-
-        <label class = field_label for = "name">Name</label>
-        <input type = "text" name="name" required = true> <br><br>
-
-        <label class = "field_label" for = "name">Email</label>
-        <input type = "email" name="email" required = true > <br><br>
-
-        <label class = "field_label" for = "message">Message</label>
-        <textarea name = "message"  required = true>
-        </textarea> <br><br>
-
-
-        <button class ="button" type = "submit"> Submit </button>
-    </form>
-
-</section>
-
-<section id = "messages">
-    <h2>Messages</h2>
-    <ul>
-
-    </ul>
-
-</section>
-
-<footer> </footer>
-<script src = js/index.js></script>
-</body>
+    <body>
+        <div>
+            <header>
+                <h2><a class = "logo" href = "#">Ki Vera Luna Pardillo</a></h2>
+                <nav>            
+                    <ul class ="nav_links">
+                        <li class = "nav_li"><a class="nav_a" href='#about'>About</a></li>
+                        <li class = "nav_li"><a class="nav_a" href='#experience'> Experience</a></li>
+                        <li class = "nav_li"><a class="nav_a" href='#skills'>Skills</a></li>
+                        <li class = "nav_li"><a class="nav_a" href="#projects">Projects</a></li>
+                        <li class = "nav_li"><a class="nav_a" href='#connect'> Connect</a></li>
+                        <li class = "nav_li"><a class="nav_a" href='#messages'>Messages</a></li>
+                    </ul>
+               
+                    <div id="mySidebar" class="sidebar sibebarbtn">
+                        <a class="nav_a closebtn" href="javascript:void(0)" onclick="closeNav()">&times;</a>
+                        <a class="nav_a" href="#welcome-section">About</a>
+                        <a class="nav_a" href="#experience">Experience</a>
+                        <a class="nav_a" href="#skills">Skills</a>
+                        <a class="nav_a" href="#projects">Projects</a>
+                        <a class="nav_a" href="#connect">Connect</a>
+                        <a class="nav_a" href='#messages'>Messages</a>
+                    </div>
+
+                    <div id="main" class="sibebarbtn">
+                        <button class="sibebarbtn sideButton" onclick="openNav()">&#9776; More</button>
+                    </div>
+
+                </nav>
+
+
+            </header> <br> <br> <br>
+            <section id = "about">
+                <h1>About</h1>
+                <p>My name is Ki Vera Luna. I am from the Philippines. My major is Computer Information System. </p>
+            </section>
+
+            <section class="list_experience" id = "experience">
```

Comment:
In the CSS you `display: flex` on this element (with the class selector `.list_experience`) but since this section only has one child (the `div` with class `.flexing`) it's not currently doing much. 

Based on the names, I'm wondering if you wanted `flex` to be applied to the unordered list below with the class `flexs`? That way all the `flex-shrink` and `flex-grow` properties you gave to the list items (`flex-box` one, two and three) would take effect